Devscord.DiscordFramework/Framework/Commands/Responses/Resources/Responses.resx

```diff
@@ -132,6 +132,12 @@
   <data name="AvailableSafeRoles" xml:space="preserve">
     <value>Dostępne role: {{roles}}</value>
   </data>
+  <data name="EmptyArgument" xml:space="preserve">
+    <value>Parametr {{arg}} jest nieprawidłowy.</value>
```

Comment:
{{arg}} nic nie mówi

Reply:
zmieniłem na botCommandArgument

Reply:
może po prostu {parameter} albo {argument}

Reply:
Argument to wartość
a Parametr to nazwa tej wartości 

np jak masz 
`-test abcd`
test to parametr 
abcd to argument

Reply:
ok